packages/jest-snapshot/src/State.ts

```diff
@@ -5,6 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import type BabelTraverse from '@babel/traverse';
```

Comment:
[Nope](https://github.com/facebook/jest/blob/master/packages/jest-snapshot/package.json#L30) :(

(which is odd because I know I checked this 🤔)

It's listed as a dev dependency

Reply:
We need the types dep

Reply:
However, this comes from `@types/babel__travese`, so we should be good to add that as the dependency 🎉